core/xchain/server/server_test.go

```diff
@@ -551,6 +557,33 @@ func deployMockErc721Contract(
 	return auth, contractAddress, erc721
 }
 
+func TestLinkedWallets(t *testing.T) {
```

Comment:
This test case will call the "sanityCheck" contract method that prints to anvil basechain